docs/targets/splunk.md

```diff
@@ -3,43 +3,51 @@
 This event target receives arbitrary [CloudEvents][ce] over HTTP and sends them to a [Splunk HTTP Event Collector][hec]
 in a [JSON format][ce-jsonformat].
 
-## Prerequisite: HTTP Event Collector Input
+## Prerequisite(s)
 
-In order to be able to use the TriggerMesh event target for Splunk, and administrator must:
+- Enable HTTP Event Collector Input
+- An HEC token
+
+Consult the [Secrets](../guides/secrets.md) guide for more information about
+how to add the HEC token as a secret.
+
+## HTTP Event Collector Input
+
+In order to be able to use the TriggerMesh event Target for Splunk, an administrator must:
 
 1. Enable the [HTTP Event Collector][hec] data input in the Splunk installation.
-1. Create a token for receiving data over HTTP.
+2. Create a token for receiving data over HTTP.
```

Comment:
I just learned this yesterday. If we number lists all as 1. like this previously was, it will auto number.